.chloggen/vcenterreceiver-update-mockserver.yaml

```diff
@@ -0,0 +1,25 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Appreciate you adding a changelog but we actually typically do not for test-only changes. I've added the `Skip Changelog` label to the PR so the check will pass if you remove it.

Reply:
@djaglowski makes sense. I'll wait for this batch of tests to finish then I'll remove this file.

Reply:
Sorry, didn't see your note. I was able to delete it directly. Should be all set.